net/host.go

```diff
@@ -24,7 +24,7 @@ const (
 	maxMessageSize = 1 << 17
 )
 
-var log = logging.Logger("host")
+var log = logging.Logger("net")
```

Comment:
I lost a lot of time on this, because I wasn't seeing necessary logs in the integration tests. We should invest some time into switching to a different logger that is more automated or figuring out a less manual way of using the existing logger.

I also don't understand what version we should be using for our existing logger.  IPFS is putting out new releases under `v2` (`github.com/ipfs/go-log/v2`), but we are still using the original version and the README in their repo refers to the original version even in the v2 releases:
https://github.com/ipfs/go-log/tree/v2.5.1/

Reply:
yeah I haven't looked into loggers recently, just been using this for a while. might be worth exploring what's out there